PreFreeSurfer/scripts/T2wToT1wDistortionCorrectAndReg.sh

```diff
@@ -18,104 +18,106 @@ FIELDMAP_METHOD_OPT="FIELDMAP"
-  [--SEPhaseNeg=<input spin echo negative phase encoding image>]
-  [--SEPhasePos=<input spin echo positive phase encoding image>]
-  [--seechospacing=<effective echo spacing of SEPhaseNeg and SEPhasePos, in seconds>]
-  [--seunwarpdir=<direction of distortion of the SEPhase images according to *voxel* axes: {x,y,x-,y-} or {i,j,i-,j-}>]
-  --t1sampspacing=<sample spacing (readout direction) of T1w image - in seconds>
-  --t2sampspacing=<sample spacing (readout direction) of T2w image - in seconds>
-  --unwarpdir=<direction of distortion of T1 and T2 according to *voxel* axes (post fslreorient2std): {x,y,z,x-,y-,z-}, or {i,j,k,i-,j-,k-}>
-  --ot1=<output corrected T1w image>
-  --ot1brain=<output corrected, brain-extracted T1w image>
-  --ot1warp=<output warpfield for distortion correction of T1w image>
-  --ot2=<output corrected T2w image>
-  --ot2brain=<output corrected, brain-extracted T2w image>
-  --ot2warp=<output warpfield for distortion correction of T2w image>
-  --method=<method used for readout distortion correction>
-
-        "${SPIN_ECHO_METHOD_OPT}"
+
+set -eu
+
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+    pipedirguessed=1
+    #fix this if the script is more than one level below HCPPIPEDIR
+    export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "${HCPPIPEDIR}/global/scripts/debug.shlib" "$@"         # Debugging functions; also sources log.shlib
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+
+opts_SetScriptDescription "Script for performing gradient-nonlinearity and susceptibility-induced distortion correction on T1w and T2w images, then also registering T2w to T1w"
+
+opts_AddMandatory '--t1' 'T1wImage' 'image' "input T1w image"
+
+opts_AddMandatory '--t1brain' 'T1wImageBrain' 'image' "input T1w brain-extracted image"
+
+opts_AddMandatory '--t2' 'T2wImage' 'image' "input T2w image"
+
+opts_AddMandatory '--t2brain' 'T2wImageBrain' 'image' "input T2w brain-extracted image"
+
+opts_AddMandatory '--t1sampspacing' 'T1wSampleSpacing' 'value (seconds)' "sample spacing (readout direction) of T1w image - in seconds"
+
+opts_AddMandatory '--t2sampspacing' 'T2wSampleSpacing' 'value (seconds)' "sample spacing (readout direction) of T2w image - in seconds"
+
+opts_AddMandatory '--unwarpdir' 'UnwarpDir' '{x,y,z,x-,y-,z-} OR {i,j,k,i-,j-,k-}' "direction of distortion of T1 and T2 according to *voxel* axes (post fslreorient2std)"
+
+opts_AddMandatory '--ot1' 'OutputT1wImage' 'image' "output corrected T1w image"
+
+opts_AddMandatory '--ot1brain' 'OutputT1wImageBrain' 'image' "output corrected, brain-extracted T1w image"
+
+opts_AddMandatory '--ot1warp' 'OutputT1wTransform' 'image' "output warpfield for distortion correction of T1w image"
+
+opts_AddMandatory '--ot2' 'OutputT2wImage' 'image' "output corrected T2w image"
+
+# opts_AddMandatory '--ot2brain' 'Ot2brain' 'image' "output corrected, brain-extracted T2w image" ## THIS ONE DOES NOT EXIST IN THE VARS?
```

Comment:
@glasserm this brain-masked T2w output argument that was in the usage was apparently never implemented.  Thoughts?

Reply:
Whoever added the usage statement put that in by mistake I think.  It wasn't in the original script.

Reply:
Done